src/js/controllers/LaunchControllers.js

```diff
@@ -68,6 +84,56 @@ angular.module('blocktrail.wallet')
     });
 
 angular.module('blocktrail.wallet')
+    .controller('OpenWalletPinCtrl', function ($scope, $rootScope, $state, $stateParams, $log, launchService, Wallet,
+                                               settingsService, $timeout) {
+        $scope.appControl = {
+            showPinInput: false,
+            showPinInputError: false,
+            pin: null,
+            result: {
+                error: null
+            }
+        };
+
+        document.addEventListener("pause", function () {
```

Comment:
should this be here in this controller?

also use `$ionicPlatform.on('pause', function() {` for consistency